embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/ReactNativeInternalInterface.kt

```diff
@@ -33,6 +33,8 @@ public interface ReactNativeInternalInterface : EmbraceInternalInterface {
 
     public fun setJavaScriptBundleUrl(context: Context, url: String)
 
+    public fun setJavaScriptBundleUrlForCodePush(context: Context, url: String, didUpdate: Boolean)
```

Comment:
Can we just the parameter and get the hosted SDK to call it with and without the parameter for code push?

Reply:
We can, it's just a fact of agreeing what's better with Damian... He preferred this way because it's explicitly clear that this call is for CodePush.

Reply:
Or alternatively, don't add the extra parameter and have two methods - one for update and one for not - so the API change isn't breaking, if you're worried about that.

Reply:
If he prefers it, we should have one method for bundle updated and one for not, and his code can pick and choose which to call instead of passing that boolean down?

Reply:
OK, I understand now - there are two pieces of new info: CAN the app update the bundle and DID the app update the bundle since last time this was called.

If that's the case, then this method should be called something like `setUpdateableJavaScriptBundleUrl` - and when this is called for the first time, set the boolean in the prefs service to true, and take the value to see if it needs updating (or if it's the first time, always update)

Reply:
updated to `setCacheableJavaScriptBundleUrl`